### PR TITLE
Update sl container for sl6 to current (build-date=20190701)

### PR DESCRIPTION
**Pull request by sl-team**

### CODE REVIEW

File: library/sl [hunk 1]
```diff
@@ -15,4 +15,4 @@ Tags: 6
 Architectures: amd64
 GitFetch: refs/heads/6
 Directory: sl6
-GitCommit: 5288d401c76c12025e0495265067ad6a912b59ce
+GitCommit: 755652f449d6cde8cf7e36fde2e9c849ac512754
```
Comment (tianon) [inline library/sl]: Shouldn't this be `098732d7f64270125a178194dbc0847bd99c1215` (https://github.com/scientificlinux/sl-docker/commit/098732d7f64270125a178194dbc0847bd99c1215, from https://github.com/scientificlinux/sl-docker/tree/6)?

Comment (jcpunk) [reply library/sl]: .... yeah, looks like the automation choked again.....

File: library/sl [hunk 2]
```diff
@@ -8,11 +8,11 @@ Tags: 7, latest
 Architectures: amd64
 GitFetch: refs/heads/7
 Directory: sl7
-GitCommit: 6f0afe07c9abcb8952c22ab76ed69d1fc757602e
+GitCommit: 4748d2717b329e8516d5ddedff8de459b1e8e20c
```
Comment (tianon) [inline library/sl]: Sorry, this one should be `0d8792816e36ec1a2f2023f48858737f6d683680` now, right? (https://github.com/scientificlinux/sl-docker/commit/0d8792816e36ec1a2f2023f48858737f6d683680 from https://github.com/scientificlinux/sl-docker/tree/7)

Comment (jcpunk) [reply library/sl]: I really should stop doing two things at once.... fixed...

**Comment by tianon** [comment]
> I really should stop doing two things at once.... fixed...

That is definitely relatable content :sweat_smile:

<details>
<summary>Diff:</summary>

```diff
diff --git a/sl_6/Dockerfile b/sl_6/Dockerfile
index 1e142c1..58e2eca 100644
--- a/sl_6/Dockerfile
+++ b/sl_6/Dockerfile
@@ -2,7 +2,7 @@
 FROM scratch
 ADD sl-6-docker.tar.xz /
 
-LABEL name="SL6 Base Image" vendor="Scientific Linux" build-date="20190603"
+LABEL name="SL6 Base Image" vendor="Scientific Linux" build-date="20190701"
 
 CMD ["/bin/bash"]
 
diff --git a/sl_6/sl-6-docker.tar.xz b/sl_6/sl-6-docker.tar.xz
index 9a70fe0..5e7a9b3 100644
Binary files a/sl_6/sl-6-docker.tar.xz and b/sl_6/sl-6-docker.tar.xz differ
diff --git a/sl_latest/Dockerfile b/sl_latest/Dockerfile
index 54217ca..2dbb988 100644
--- a/sl_latest/Dockerfile
+++ b/sl_latest/Dockerfile
@@ -2,7 +2,7 @@
 FROM scratch
 ADD sl-7-docker.tar.xz /
 
-LABEL name="SL7 Base Image" vendor="Scientific Linux" build-date="20190603"
+LABEL name="SL7 Base Image" vendor="Scientific Linux" build-date="20190701"
 
 CMD ["/bin/bash"]
 
diff --git a/sl_latest/sl-7-docker.tar.xz b/sl_latest/sl-7-docker.tar.xz
index 5fe2d84..2c3eeff 100644
Binary files a/sl_latest/sl-7-docker.tar.xz and b/sl_latest/sl-7-docker.tar.xz differ
````

</details>

**Comment by tianon** [comment]
LGTM

Build test of #6194; 2c522c6b; `amd64` (`sl`):

```console

$ bashbrew build sl:7
Building bashbrew/cache:c772966692ba9d67fed27f7315938967ea9e46090ae99a85ac7b39580e79f682 (sl:7)
Tagging sl:7
Tagging sl:latest

$ test/run.sh sl:7
testing sl:7
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build sl:6
Building bashbrew/cache:67f0ea5dddf4564c00f8c4adf998296fe71d40fd88ec398236197fc756a7b28c (sl:6)
Tagging sl:6

$ test/run.sh sl:6
testing sl:6
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**